api/v1alpha3/cluster_types.go

```diff
@@ -36,6 +38,9 @@ type ClusterSpec struct {
 	// +optional
 	Paused bool `json:"paused,omitempty"`
 
+	// PostApplyAddons is a list of Secrets in YAML format to be applied to remote clusters.
+	PostApplyAddons []PostApplyAddon `json:"postApplyAddons,omitempty"`
```

Comment:
```suggestion
	PostInitComponents []ComponentRef `json:"postInitComponent,omitempty"`
```

This is only a suggestion, the `Addons` word is overloaded, and I would like to find something easier to understand. `PostInit` suggests that the components are going to be applied only after the Cluster has been initialized, in this case we know that we'll wait for the control plane.